docs/source/contents/4_draft_mapping.rst

```diff
@@ -1883,8 +1920,8 @@ None type
 Use Case
 ^^^^^^^^
 
-Several elements contain unintentional null values. There are five within Tennessee Documentary History. Additional null
-subjects include vpmoore:133 and adams:76. Most of roth seems to have null mods:subject/mods:name/mods:namePart values.
+Several :code:`subject` elements contain unintentional null values. There are five within Tennessee Documentary History. Additional null
+:code:`subject`\ s include vpmoore:133 and adams:76. Most of roth seems to have null :code:`subject`\ /:code:`name`\ /:code:`namePart` values.
```

Comment:
Curious:  why not :code:`subject/name/namePart` ?

Reply:
This is where it's hard to change text values of single elements to code. If we are going to keep with making these into code I think we do need to spell out the Xpath. It gets a bit lengthy, but it's very clear this way.

Reply:
(-_-) I don't have any idea what I was thinking here. PEBKAC.